### ISSUE TITLE
criterion = nn.CrossEntropyLoss(ignore_index=args.ignore_label).cuda()？

### ISSUE THREAD

**Issue by swzaaaaaaa** [CVMI-Lab/PAConv#28]
请问，这个ignore_index = 255 是什么意思？

**Comment by Dingry** [comment]
Hi, ignore index is the ignored label id that has no contribution to gradient computing. 255 is the most common value.
